### DISCUSSION TITLE
Profanity filter / Word Replacement Feature in forum?

### DISCUSSION THREAD

**Discussion by gnossosp** [DNNCommunity/Dnn.CommunityForums#904] [Q&A] [answered]
Is there a way to customize a word/profanity filter for the forum?
In addition to the usual potty mouth words I would like to use custom words/replacements for example if someone types Biden it replaces text with Dark Brandon.
 I see the "Filters" item in the control panel but I'm not sure if that is it.  It is blank and there is no response when "Load Default Filters"

**Comment by johnhenley** [comment, marked as answer]
@gnossosp The "Load Default Filters" is fixed in 8.1, which is coming out soon I hope :)

For now. you can manually add:
![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/e9a6d377-6e0d-46fc-aa16-e589486258ef)
![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/0b121a5d-b7c1-4416-bd47-36fbe90f22fa)

![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/44a8093c-e54d-400d-bc0f-1db6437814c0)
When the topic is saved, the text is replaced:
![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/06273e51-df22-407e-abe0-7d7bc6c8ac00)

**Comment by gnossosp** [comment]
Muhahaha I am going to have so much fun with this!  Thanks
